pom.xml

```diff
@@ -62,18 +62,15 @@
   </pluginRepositories>
 
   <properties>
-    <!--
-      explicitly specifying the latest version here because one we get from the parent POM
-      tends to lag behind a bit
-    -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
```

Comment:
imho you can update jenkins core version to 1.565.3 that is previous LTS branch version and contains exactly the same hpi-plugin version.